content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -317,6 +317,38 @@ product form.
 
 .. _sales/advanced-price-rules:
 
+Discounts
+=========
+
+The :guilabel:`Discounts` feature allows the ability to set a discount or increase the price
+on *individual items* in a sales order. This is calculated as a percentage of that product's sales
+price.
+
+To access discounts, navigate to the *Sales* app setting page
+(:menuselection:`Sales app --> Configuration --> Settings`), and click the :guilabel:`Discounts`
+checkbox, then click :guilabel:`Save`.
+
+.. image:: pricing/discounts-option.png
+   :align: center
+   :alt: Click the checkbox labeled `Discounts` to activate Discounts.
+
+To set a discount after discounts have been activated, navigate to the list of sales orders by
+going to :menuselection:`Orders --> Orders` at the top of the page. Once there, click on the
+desired sales order from the list.
+
+In the :guilabel:`Order Lines` section of the sales order form, a new heading will appear labeled
```

Comment:
- **Order lines** doesn't get a GUI I don't think, as it's not text or a label on the UI.
- sales order --> quotation (based on previous comment)
- heading --> column heading (specificity)

```suggestion
In the order lines section of the quotation form, a new column heading will appear labeled
```